resources/aws/common.go

```diff
@@ -5,5 +5,6 @@ import (
 )
 
 type AWSEntity struct {
-	Clients awsutil.Clients
+	Clients     awsutil.Clients
+	HostClients awsutil.Clients
```

Comment:
Same here. There should be a separate block for guest/host magic and the clients should be in there accordingly.